test/recipes/95-test_external_oqsprovider_data/oqsprovider-externalinterop.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
This is essentially an file that does 'exit 0'..
A coment and exit 0 seems sufficient.

Reply:
That is correct and I considered doing this. But what changed my mind was the goal to provide pointers as to what's the original test and where to check for details & how to fix this for good. Thus I simply moved the original code from oqs-provider facilitating just the same "quick fix" to work around the CF interop server code change until the OQS code catches up again. This stuff should be removed in 1-3 weeks and only is a quick fix to get OpenSSL CI back to green. Hence I prefer to keep the whole context also here.